### PR TITLE
ankurguptajec@gmail.com

### PR DESCRIPTION
**Pull request by ankurguptajec**
Example Code for the article "Using Multiple Cache Managers in Spring"

### CODE REVIEW

File: spring-caching/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+#spring.h2.console.enabled=false
```
Comment (maibin) [inline spring-caching/src/main/resources/application.properties:5]: we have this commented out and few lines later it appears again

File: spring-caching/pom.xml [hunk 2]
```diff
@@ -15,6 +15,10 @@
     </parent>
 
     <dependencies>
+    	<dependency>
+			<groupId>org.springframework.boot</groupId>
```
Comment (maibin) [inline spring-caching/pom.xml:19]: let's clean up formatting in here

File: spring-caching/pom.xml [hunk 3]
```diff
@@ -40,6 +44,19 @@
             <artifactId>spring-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+    		<groupId>com.github.ben-manes.caffeine</groupId>
+    		<artifactId>caffeine</artifactId>
+    	</dependency>
+    	<dependency>
+			<groupId>com.h2database</groupId>
```
Comment (maibin) [inline spring-caching/pom.xml:52]: same here

File: spring-caching/src/main/java/com/baeldung/multiplecachemanager/config/MultipleCacheManagerConfig.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+//import org.springframework.context.annotation.Primary;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+
+@Configuration
+@EnableCaching
+public class MultipleCacheManagerConfig extends CachingConfigurerSupport {
+
+    @Bean
+    //@Primary
```
Comment (maibin) [inline spring-caching/src/main/java/com/baeldung/multiplecachemanager/config/MultipleCacheManagerConfig.java:20]: why this one is commented out? if that's purposeful, then let's keep it as it is

File: spring-caching/src/test/java/com/baeldung/multiplecachemanager/MultipleCacheManagerIntegrationUnitTest.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    private OrderDetailBO orderDetailBO;
+
+    @Autowired
+    private CacheManager cacheManager;
+
+    @Autowired
+    private CacheManager alternateCacheManager;
+
+    @Test
+    public void whenCallGetOrderDetail_thenDataShouldBeInCaffieneCacheManager() {
```
Comment (maibin) [inline spring-caching/src/test/java/com/baeldung/multiplecachemanager/MultipleCacheManagerIntegrationUnitTest.java:35]: format for BDD is givenX_whenY_thenZ

File: spring-caching/src/main/java/com/baeldung/multiplecachemanager/config/MultipleCacheManagerConfig.java (hunk 4 above)
Comment (ankurguptajec) [reply spring-caching/src/main/java/com/baeldung/multiplecachemanager/config/MultipleCacheManagerConfig.java:20]: Yes, It is purposeful.

File: spring-caching/src/main/resources/application.properties (hunk 1 above)
Comment (ankurguptajec) [reply spring-caching/src/main/resources/application.properties]: Fixed this

File: spring-caching/pom.xml (hunk 2 above)
Comment (ankurguptajec) [reply spring-caching/pom.xml:20]: Done

File: spring-caching/pom.xml (hunk 3 above)
Comment (ankurguptajec) [reply spring-caching/pom.xml:53]: Done

File: spring-caching/src/test/java/com/baeldung/multiplecachemanager/MultipleCacheManagerIntegrationUnitTest.java (hunk 5 above)
Comment (ankurguptajec) [reply spring-caching/src/test/java/com/baeldung/multiplecachemanager/MultipleCacheManagerIntegrationUnitTest.java]: Done

**Pull request merged**